src/main/java/frc/robot/subsystems/Turret.java

```diff
@@ -46,4 +53,15 @@ public void stop(){
   public double getAngle() {
     return (m_pot.getAverageVoltage() / Constants.AIO_MAX_VOLTAGE) * Constants.TURRET_POT_SCALE_VALUE * Constants.TURRET_RATIO;
   }
+  
+  @Override
+  public void useOutput(double output, double setpoint) {
+    // Use the output here
+  }
+
+  @Override
+  public double getMeasurement() {
+    // Return the process variable measurement here
+    return 0;
+  }
```

Comment:
Functionality of the already existing move turret commands should be implemented here

Reply:
Fix in commit f018a89